### ISSUE TITLE
Vulkan Support?

### ISSUE THREAD

**Issue by jeffbdavenport** [calebuharrison/CrystGLFW#7]
I couldn't find anywhere in the code or documentation about how to use Vulkan, which GLFW currently supports. Is this code out of date?

https://www.glfw.org/docs/latest/vulkan_guide.html

**Comment by calebuharrison** [comment]
Yes, CrystGLFW does not currently provide an idiomatic interface for GLFW's Vulkan API.  I don't have plans to add it myself, but I would accept a PR.

**Comment by da1nerd** [comment]
Related to this, it appears there is some effort to support vulkan in [this fork](https://github.com/nilium/LibGLFW/tree/v3.3) of the GLFW bindings, specifically [this file](https://github.com/nilium/LibGLFW/blob/v3.3/src/lib_glfw/vulkan.cr).

## Edit
After some more looking around I discovered [another fork](https://github.com/malte-v/glfw.cr) that is quite a bit further along (at least in raw bindings). Here are some dedicated [vulkan bindings](https://github.com/malte-v/vulkan.cr).
